Dockerfile.std

```diff
@@ -0,0 +1,21 @@
+# Generated by: tyk-ci/wf-gen
+# Generated on: Wed 17 Feb 17:51:15 UTC 2021
+
+# Generation commands:
+# ./pr.zsh -title use nfpm for std docker #2 -branch goreleaser/more -p
+# m4 -E -DxREPO=tyk
+
+FROM debian:buster-slim
+RUN apt-get update \
```

Comment:
Doesn't this need Python and friends as in the existing image?

Reply:
Thanks!